Software/spartan/src/sensors/ads1115.cpp

```diff
@@ -1,8 +1,21 @@
 #include "ads1115.h"
-/*
-spartan::ADS1115::ADS1115(int bus, uint8_t address) {}
 
-bool spartan::ADS1115::pollData(spartan::DataPacket &dp) { return 0; }
+spartan::ADS1115::ADS1115(int bus, uint8_t address)
+    : Sensor(bus, address), m_i2c(bus, true) 
+{}
 
-void spartan::ADS1115::printValues() const {}
-*/
+const char* spartan::ADS1115::name() const 
+{ 
+    const char * str = "ADS1115";
+    return str;
+};
+
+void spartan::ADS1115::printSensorInfo() {}
+
+int spartan::ADS1115::powerOn() { return 0; }
+
+int spartan::ADS1115::powerOff() { return 0; }
+
+int spartan::ADS1115::poll(MasterDataPacket &dp) { return 0; }
```

Comment:
This should be renamed to `pollData`, as this is the new interface for the `Sensor` abstract base class. See #48 for details.